app/uk/gov/hmrc/tai/util/constants/EditIncomePayPeriodConstants.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tai.util.constants
+
+trait EditIncomePayPeriodConstants {
+  val MONTHLY = "monthly"
```

Comment:
Since these pop up in a few places, we should consider making this a sum type ADT which will allow the compiler to perform exhaustive type checking of these options.